CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.0.0] - 2022-04-01
+### Changed
+- hyphens are no longer allowed in variable names in yaml files
+- an empty config file can no longer be created via the configure method
+- data bucket config name modified to reflect the fact that it denotes the bucket where the anonymised exports will be kept
```

Comment:
Could we mark this as a backwards incompatible change?

Reply:
I can add that, I just thought that making it a major version upgrade would do that so that's why I didn't add it initially

Reply:
Yeah, it tells you that the version contains breaking changes, but it's nice to tell the users which of the changes are breaking, so they don't have to figure out themselves ;)